src/civis/resources/_resources.py

```diff
@@ -51,11 +47,12 @@
 
 
 def exclude_resource(path, api_version):
-    if api_version == "1.0":
-        include = any(path.startswith(x) for x in BASE_RESOURCES_V1)
-    else:
-        include = True
-    return not include
+    # TODO: api_version is not used here.
+    #   Dropping it would affect upstream code, including civis.APIClient.
+    #   We may need to deprecate api_version in the future.
```

Comment:
Deprecating the `api_version` param across the board, if done at all, would come from a future PR.